pyproject.toml

```diff
@@ -38,9 +38,19 @@ dependencies = [
 dynamic = ["version"]
 license.file = "LICENSE"
 readme = "README.md"
-requires-python = ">=3.11"
+requires-python = ">=3.11,<3.13"
```

Comment:
The current best practice appears to be never to cap python version for libraries:
https://iscinumpy.dev/post/bound-version-constraints/#tldr

I would argue it probably makes sense here too...

Reply:
This was probably a hang over from epics(?) not supporting 3.13 and can be removed now, especially if we're using `.python-version`. I'm pretty sure `uv` ignores upper bounds anyway.